src/generators/genlua.ml

```diff
@@ -334,6 +403,21 @@ let rec gen_call ctx e el in_value =
 			List.iter (fun p -> print ctx ","; gen_value ctx p) params;
 			spr ctx ")";
 		);
+		| TField( _, FStatic({ cl_path = ([], "Std") }, { cf_name = "string" }) ), [e]
```

Comment:
Here's where I'm hacking into the Std.string behavior to find multireturns and override their string rendering.